presentation/src/main/java/com/knocklock/presentation/lockscreen/LockScreen.kt

```diff
@@ -121,13 +123,23 @@ fun UnLockSwipeBar(
             repeatMode = RepeatMode.Reverse
         )
     )
-    LaunchedEffect(swipeableState.targetValue) {
-        if (swipeableState.targetValue == 1) {
-            userSwipe()
+
+    val targetValue by remember(swipeableState) {
+        derivedStateOf {
+            swipeableState.targetValue == 1
         }
     }
-    LaunchedEffect(swipeableState.progress.fraction) {
-        updateTransitionState(swipeableState.progress.fraction == 1f)
+    val fraction by remember(swipeableState) {
+        derivedStateOf {
+            swipeableState.progress.fraction == 1f
+        }
+    }
+
+    if (targetValue) {
+        userSwipe()
+    }
+    if (fraction) {
+        updateTransitionState(fraction)
```

Comment:
이 부분은 기존 코드를 유지하려면 true 일때만 호출하지 않고 false일 때도 불러줘야 할 것 같은데 맞을까요? 흠 이렇게하면 변경된 의미가 없게 되는 걸까요 fraction 변수를 넘겨줌으로 skip이 recomposition 시에 skip이 될 것 같은데요

```suggestion
     updateTransitionState(fraction)
```

Reply:
@jaeryo2357 맞네영 fraction이 false일때도 불러줘야해요 !!

Reply:
@jaeryo2357 하지만 다음에 성능 문제를 해결하기 위해서 조금 코드를 축소화하려고합니다.[b16aa67](https://github.com/Knock-Lock/Knock-Lock/pull/101/commits/b16aa67e10fbb03b8988035f167fea8c7a1d50f2) 당장은 필요가 없을꺼같아서 제거하도록 하겠습니다